### PR TITLE
PERF: add a poor man's profiler

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: src/main/kotlin/org/rust/utils/RsStopWatch.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            override fun run() {
+                println("${this@RsStopWatch.name}: ${totalNs.get() / 1_000_000} ms")
+            }
+        })
+    }
+
+    fun <T> measure(block: () -> T): T {
+        var result: T? = null
+        totalNs.addAndGet(measureNanoTime { result = block() })
+        return result!!
```
Comment (mkaput) [inline src/main/kotlin/org/rust/utils/RsStopWatch.kt]: I think it should be good to note that returning `null` from block will result in NPE

Comment (matklad) [inline src/main/kotlin/org/rust/utils/RsStopWatch.kt]: Excellent catch!

**Comment by matklad** [comment]
bors r=vlad20012

**Pull request merged**